### DISCUSSION TITLE
02/022-%E7%AC%AC%E4%BA%8C%E5%8D%81%E4%BA%8C%E7%AB%A0-%E5%8F%98%E5%8C%96%E7%9A%84%E9%A3%8E%E5%90%91.html

### DISCUSSION THREAD

**Discussion by giscus** [liuli-moe/to-the-stars#25] [General]
02/022-%E7%AC%AC%E4%BA%8C%E5%8D%81%E4%BA%8C%E7%AB%A0-%E5%8F%98%E5%8C%96%E7%9A%84%E9%A3%8E%E5%90%91.html
序言
https://tts.liuli.moe/02/022-%E7%AC%AC%E4%BA%8C%E5%8D%81%E4%BA%8C%E7%AB%A0-%E5%8F%98%E5%8C%96%E7%9A%84%E9%A3%8E%E5%90%91.html

**Comment by bixudamajiang** [comment]
作者挂猎枪的技术可以的
